### DISCUSSION TITLE
Problems with quasar.conf.js

### DISCUSSION THREAD

**Discussion by bv-bjj** [quasarframework/quasar#11826] [General - Components / Directives / etc] [answered]
Hello everyone,
I'm new to quasar and in the way to develop a new app I'm finding some problems related to quasar.conf.js. Basically, I don't have this file. I suspect that is because I've installed quasar after installing vue cli. So, I'm looking for this file or a different way to configure general things of Quasar. Now, I was inspecting node modules folderes related to Quasar but I couldn't find anything. 

I only am trying to change the default font.

If there's a way of solving this I would really appreciate. Otherwise, I may install quasar cli directly.

Thanks in advance,

**Comment by rstoenescu** [comment, marked as answer]
The quasar.conf.js file is available only for Quasar CLI projects.

**Reply by hawkeye64** [reply]
There's lots of information for configuring quasar, in the online docs, when using the Vue CLI.
